portfolio/src/main/webapp/script.js

```diff
@@ -75,7 +75,12 @@ function createCommentElement(comment) {
     commentElement.remove();
   });
 
-  commentElement.appendChild(titleElement);
+  if (comment.imageUrl != 'null') {
```

Comment:
See my comment https://github.com/aannie-y/step/pull/10#discussion_r549509457

If not fixed at least set a constant.

Reply:
Fixed.